packages/components/note/src/Note.tsx

```diff
@@ -0,0 +1,100 @@
+import { cx } from 'emotion';
+import React from 'react';
+import { Primitive } from '@contentful/f36-core';
+import type { CommonProps, ComponentVariant } from '@contentful/f36-core';
+import { styles } from './Note.styles';
+import { IconButton } from '@contentful/f36-components';
+import {
+  CheckCircle,
+  Close,
+  ErrorCircle,
+  InfoCircle,
+  Warning,
+} from '@contentful/f36-icons';
+
+const icons = {
+  primary: InfoCircle,
+  positive: CheckCircle,
+  negative: ErrorCircle,
+  warning: Warning,
+};
+
+export type NoteVariant = Exclude<ComponentVariant, 'secondary'>;
+export interface NoteProps extends CommonProps {
+  /**
+   * Determines style variation of Note component
+   */
+  variant?: NoteVariant;
+  /**
+   * Sets title in the Note
+   */
+  title?: string;
+  /**
+   * Children of Note
+   */
+  children?: React.ReactNode;
+  /**
+   * Flag for showing close button
+   */
+  hasCloseButton?: boolean;
+  /**
+   * Callback for handling closing
+   */
+  onClose?: Function;
+}
+
+const _Note = (props: NoteProps, ref) => {
+  const {
+    children,
+    className,
+    hasCloseButton,
+    variant = 'primary',
+    onClose,
+    style,
+    testId = 'cf-ui-note',
+    title,
+    ...otherProps
+  } = props;
+
+  const Icon = icons[variant!]; // eslint-disable-line @typescript-eslint/no-non-null-assertion
+  if (!Icon) {
+    throw new Error(`Intent ${variant} is not supported in Note component.`);
+  }
+
+  return (
+    <Primitive
+      as="div"
+      style={style}
+      className={cx(styles.note({ variant, hasCloseButton }), className)}
+      testId={testId}
+      ref={ref}
+      {...otherProps}
+    >
+      <div className={styles.icon}>
+        <Icon variant={variant} size={title ? 'medium' : 'small'} />
+      </div>
+      <div className={styles.info({ hasCloseButton })}>
+        {title && <div className={styles.title}>{title}</div>}
+        <div className={styles.content}>{children}</div>
```

Comment:
Should these elements use typography? title should be `h2` and probably content is `p`?

Reply:
I don't know if we can determine the heading level? Paragraph makes sense to me.

Reply:
I probably would not use h2 for the title since we don't always know the context, would probably be okay with p but I don't really know what is the design/copy guideline for the content and usecases...
Happy to change whatever you feel like would be an improvement to the experience/accessibility in this case.

Reply:
yeah, you're right, I thought it's Notification 🙃 Should we expose the title element in `as` prop then, and require it to be set?

Reply:
If we use `article` as the root element we should be able to safely use `h1` for the title. How about that? I'd prefer some sane defaults to forcing people to set a header level.

Reply:
@mshaaban0 how would you propose it looks like when using? 
`<Note title="this is title" as="h3" hasCloseButton>children go here</Note>` ?

Reply:
By the style guide it would probably be something like `<Note title="this is title" titleProps={{ as: "h3" }} hasCloseButton>children go here</Note>`

Reply:
The `article` tag is actually a good idea, as it will present it as self contained content, but I believe title should be `h2`, as `h1` should be preserved only to page title.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
https://www.w3.org/WAI/tutorials/page-structure/headings/

Reply:
so.. the Primitive would be the article and I use h2 for the title, does that sound okay for everyone?

Reply:
I believe `article` also signifies a "new" heading context but I think it's open for interpretation. `h2` works for me 👍